reviews/2022-cdk8s-annual.md

```diff
@@ -0,0 +1,101 @@
+## Background
+
+CDK For Kubernetes (cdk8s) is an open-source software development framework for defining Kubernetes 
+applications and reusable abstractions. Users leverage familiar programming languages and rich object-oriented APIs to 
+create standard Kubernetes manifests which can be applied to any Kubernetes cluster.
+
+CDK8s joined the CNCF on November 10, 2022.
```

Comment:
@iliapolo is this date accurate? [CNCF site](https://www.cncf.io/projects/cdk-for-kubernetes-cdk8s/) shows November 10, 2020

Reply:
Whoops, good catch! I'll change.